src/namespaces.ts

```diff
@@ -0,0 +1,6 @@
+import namespace from '@rdfjs/namespace';
+
+export const hydra = namespace('http://www.w3.org/ns/hydra/core#');
+export const owl = namespace('http://www.w3.org/2002/07/owl#');
+export const rdf = namespace('http://www.w3.org/1999/02/22-rdf-syntax-ns#');
+export const schema = namespace('http://schema.org/');
```

Comment:
Think I remember seeing a library that could replace this. Useful to have one that knows the terms to avoid typos.

Reply:
`@zazuko/rdf-vocabularies` does that to some extent

Reply:
[`@ontologies/schema`](https://www.npmjs.com/package/@ontologies/schema) etc was what I was thinking of. Just tried but ran into https://github.com/ontola/ontologies/issues/3.